careers/blocks/featured-card/featured-card.css

```diff
@@ -0,0 +1,121 @@
+.section.featured-card-container {
+  padding-top: var(--layout-size-xl);
+  padding-bottom: var(--layout-size-xl);
+}
+
+.featured-card-wrapper-no-background {
+  padding-bottom: var(--layout-size-xs) !important;
+  background-color: var(--static-color-black) !important;
+}
+
+.featured-card-wrapper {
+  padding-bottom: var(--layout-size-xs) !important;
+  background-color: rgba(0 0 0 / 80%);
+}
+
+.featured-card .featured-card-background {
+  box-sizing: border-box;
+  overflow: hidden;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.featured-card .featured-card-background > div > picture {
+  position: absolute;
+  z-index: -4;
```

Comment:
Why `-4`?

Reply:
Updated. Reason was that when used in other Caesars pages, previously some stacking happened with other blocks and it needed that z-index value to keep the "correct" order (as in the existing webpage).